core/src/test/scala/org/apache/spark/metrics/MetricsSystemSuite.scala

```diff
@@ -42,7 +43,7 @@ class MetricsSystemSuite extends SparkFunSuite with BeforeAndAfter with PrivateM
     val metricsSystem = MetricsSystem.createMetricsSystem("default", conf, securityMgr)
     metricsSystem.start()
     val sources = PrivateMethod[ArrayBuffer[Source]]('sources)
-    val sinks = PrivateMethod[ArrayBuffer[Source]]('sinks)
+    val sinks = PrivateMethod[ArrayBuffer[Sink]]('sinks)
```

Comment:
Is this actually a bug fix?

Reply:
I thought the same, but tests are passing. So decided as nit fixes.